src/main/java/net/dkamps18/channelpoints/Handler/CommandHandler.java

```diff
@@ -39,13 +37,13 @@ public boolean run(CommandSender sender, Command cmd, String label, String[] arg
 		Player p = (Player) sender;
 		switch (args[0].toLowerCase()) {
 			case "connect": {
-				ResultSet res = this.plugin.dbu.get_user_data(p.getUniqueId().toString());
+				ResultSet res = this.plugin.databaseUtil.getUserData(p.getUniqueId().toString());
 				try {
 					if (res.next()) {
-						this.plugin.PubSub.addStream(p, res.getString("twitchid"), res.getString("authtoken"));
+						this.plugin.pubSubHandler.addStream(p, res.getString("twitchid"), res.getString("authtoken"));
 					} else {
 						p.sendMessage(ChatColor.RED + "No authentication details found");
-						p.spigot().sendMessage(this.authmsg);
```

Comment:
You can't send a Text component with the normal api

Reply:
Doable with PaperSpigot, I just assumed that was the usual but I can revert this.